packages/Maleo.js/src/build/static/static.ts

```diff
@@ -0,0 +1,39 @@
+// tslint:disable:no-console
+import fs from 'fs';
+import path from 'path';
+import { StaticPages } from '@interfaces/build/IWebpackInterfaces';
+import { render } from '@server/render';
+import { BUILD_DIR, STATIC_BUILD_DIR } from '@constants/index';
+
+/**
+ * acquire corresponding pages which want to be exported as static html
+ * render App, Wrap, & Document for each page
+ * renderToString & dump it to file
+ */
+export const buildStatic = async (staticPages: StaticPages, dir: string) => {
+  const cwd: string = path.resolve(dir);
+  const assetDir: string = path.resolve(cwd, BUILD_DIR, 'client');
+  const staticDir: string = path.resolve(cwd, STATIC_BUILD_DIR);
+  if (!fs.existsSync(staticDir)) {
```

Comment:
Isn't `fs.existsSync` is going to be deprecated soon?
I found an article hope it helps https://stackoverflow.com/questions/31799274/node-js-fs-exists-will-be-deprecated-what-to-use-instead